### PR TITLE
Do not allow empty values for the badge title and custom CSS/JS scripts

### LINKED ISSUE #4630: Preview bar broken if empty backend parameters in config.xml
### Affected version(s)

4.13.4

### Description

Acutally i had two similar installations. On the PROD site the Preview bar was broken after updating to 4.13.4. On DEV site everything was fine. After rerolling to 4.12.7 the error occured here too.

Bildschirmfoto 2022-05-06 um 09 27 43
Dev Konsole

I figured out the reason of this problem. Empty backend parmeters in config.yml turned out to be the problem:

**DEV**
```
backend:
        attributes:
            app-name: JBZ Testserver
        badge_title: JBZ Testserver
        custom_css:
            - files/backend/custom.css

```

**PROD**
```
  backend:
        attributes:
            app-name:
        badge_title:
        custom_css:
            - files/backend/custom.css
```

After removing empty lines "attributes, app-name, badge_title" everything is working fine again.

```
 backend:
        custom_css:
            - files/backend/custom.css
```

Topic Contao-Forum: https://community.contao.org/de/showthread.php?83068-quot-Error-while-loading-the-preview-bar-quot-Nach-Update-auf-4-13-4&p=558728&viewfull=1#post558728

### PR DESCRIPTION
**Pull request by leofeyer**
Fixes #4630

### CODE REVIEW

**Review by ausi — APPROVED** [review]

File: core-bundle/src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -602,16 +602,19 @@ static function (array $attributes): array {
...
                     ->info('Adds custom JavaScript files to the back end.')
                     ->example(['files/backend/custom.js'])
+                    ->cannotBeEmpty()
                     ->scalarPrototype()->end()
                 ->end()
                 ->scalarNode('badge_title')
                     ->info('Configures the title of the badge in the back end.')
                     ->example('develop')
+                    ->cannotBeEmpty()
                     ->defaultValue('')
```
Comment (fritzmg) [inline core-bundle/src/DependencyInjection/Configuration.php:618]: I think this won't work, as the default is an empty value.

Comment (leofeyer) [reply core-bundle/src/DependencyInjection/Configuration.php:618]: Have you tested it? Because it does work on my system.

Comment (leofeyer) [reply core-bundle/src/DependencyInjection/Configuration.php:618]: @fritzmg /ping

Comment (fritzmg) [reply core-bundle/src/DependencyInjection/Configuration.php:618]: Yes, it does seem to work.

**Review by fritzmg — APPROVED** [review]

**Pull request merged**